StreamingKit.podspec

```diff
@@ -1,11 +1,11 @@
 Pod::Spec.new do |s|
   s.name         = "StreamingKit"
-  s.version      = "2.4.23"
+  s.version      = "2.4.24"
   s.summary      = "A fast and extensible audio streamer for iOS and OSX with support for gapless playback and custom (non-HTTP) sources."
   s.homepage     = "https://github.com/tumtumtum/StreamingKit/"
   s.license      = 'MIT'
   s.author       = { "Thong Nguyen" => "tumtumtum@gmail.com" }
-  s.source       = { :git => "https://github.com/GlobalRadio/StreamingKit.git", :tag => "guac_2.4.23"}
+  s.source       = { :git => "https://github.com/GlobalRadio/StreamingKit.git", :tag => "guac_2.4.24"}
```

Comment:
Why do we have to specify tag in here?

Reply:
You have to specify the location where it can find the library. 
https://guides.cocoapods.org/syntax/podspec.html#source
I guess we could also reference it like 
`"guac_#{spec.version}"`